braindecode/classifier.py

```diff
@@ -221,7 +221,7 @@ def predict_proba(self, X):
 
         """
         y_pred = super().predict_proba(X)
-        if self.cropped:
+        if self.cropped and len(y_pred.shape) == 3:
```

Comment:
Please add some comment here why this happens! And what we expect etc....

Reply:
```suggestion
        if self.cropped and len(y_pred.shape) == 3:
```

```suggestion
# Normally, we have to average the predictions across crops/timesteps
# to get one prediction per time window/trial
# In the cropped scoring , we may have averaged these predictions 
# already, hence the check if len(y_pred.shape) == 3:
        if self.cropped and len(y_pred.shape) == 3:
```

Reply:
this is what I left for later (together with the comments to do) if the changes from this PR works with your example. @robintibor have you checked if ti works with your example with variable trial length?

Reply:
I checked it in my mind and it seemed to work! Now also checked in reality and also worked! :)

Reply:
so do you have capacity to finish it? @sliwy  Otherwise I can have a try :)

Reply:
@robintibor I can take care of it today or tomorrow evening. If you need it faster feel free to finish that. :)

Reply:
sounds good! should be quick enough! :)